package.json

```diff
@@ -90,6 +98,8 @@
       "packages/fluentui/*"
     ],
     "nohoist": [
+      "packages/web-components/@storybook",
```

Comment:
explicitly opting out from any single version policy SB changes for packages that we don't own